### ISSUE TITLE
Pure javascript

### ISSUE THREAD

**Issue by d00001** [lindell/JsBarcode#29]
Docs says this plugin can be used in "pure" javascript. It doesnt look like plugin attaches anything to window object. How can I access the plugin in this mode?

**Comment by nicovanzyl** [comment]
@d00001 Did you ever get it working correctly with just vanilla js?

**Comment by lindell** [comment]
Late response. But it seems like nothing is wrong. Tried to generate with.
```
JsBarcode(document.getElementById("barcode1"), "hello");
```

The problem you had is probably that you tried
```
JsBarcode("#barcode1", "hello");
```
I try to add so that both those ways are possible to do

**Comment by lindell** [comment]
Fixed with 2d0eea5473037b62508937581b38590d786e729b
